docs/Options.md

```diff
@@ -78,6 +78,9 @@ Directory is always shown and truncated to the value of `SPACEFISH_DIR_TRUNC`. W
 | `SPACEFISH_DIR_TRUNC_REPO` | `true` | While in `git` repo, show only root directory and folders inside it |
 | `SPACEFISH_DIR_COLOR` | `(set_color --bold cyan)` | Color of directory section |
 | `SPACEFISH_DIR_PREFIX` | `in·` | Prefix before current directory |
+| `SPACEFISH_DIR_LOCK_SHOW` | `true` | Show directory write-protected symbol |
+| `SPACESHIP_DIR_LOCK_SYMBOL` | ![·🔒](https://raw.githubusercontent.com/matchai/spacefish/master/images/padlock-icon.png) | The symbol displayed if directory is write-protected (requires powerline patched font) |
```

Comment:
It's better not to add images to repository because they're cloned during installation. Instead we could upload it to comments and then use that link in the doc.

Reply:
![46248218-4af95d80-c434-11e8-8e25-595d792503f1](https://user-images.githubusercontent.com/11844760/47611530-7bf99c00-da8d-11e8-95da-f4ec1f23203a.png)
🎉